README.md

```diff
@@ -1,25 +1,65 @@
+Designed specifically for areas with little to no internet, our application works fully offline and uses limited storage on smartphones. Additionally, our app functions in three main languages: French, English, and Chadian Arabic, with options to add more local languages. Our vision is a better future for African children through improved math skills, and this starts with parents and their children using Growth+.
 
-## Meet the Team
+![growth+](https://github.com/ucfcs/GrowthPlus/assets/45129978/fce91af6-a696-442f-9e75-c6f7cef84397)
 
-### Project Manager
-* Ruth Shryock 
+## Getting Started
 
-### UI Layer
-* Victor Yue
+All you need to work on the project is to set up an android development environment, follow the official android documentation and make sure to check the system requirements:  
+https://developer.android.com/studio/install.
 
-### Data/Domain Layer
-* Julio Alva
-* Mack Orooji
-* Leticia Barbosa
+## Fork the repository
+
+<img align="right" width="300" src="https://github.com/ucfcs/GrowthPlus/assets/45129978/4a06cd1c-b08b-4653-8346-c2c368252ac2" alt="fork this repository" />
+
+- Fork the repository associated with the project to your local GitHub organization/account. This means that you will have a copy of the repository under  **your-GitHub-username/repository-name**.
+
+<br />
+<br />
+<br />
+<br />
+
+## Clone the repository and run the project
+
+- Now clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on the code button and then click the _copy to clipboard_ icon.
+
+<img width="300" height= "300" src="https://github.com/ucfcs/GrowthPlus/assets/45129978/b7cfd290-f8e0-446a-bd2d-6eaf4ef43603" alt="clone this repository" />
+
+<br />
+<br />
+
+- Open Android studio and click on `Get from VCS`
+
+<img width="600" src="https://github.com/ucfcs/GrowthPlus/assets/45129978/1cc36dcc-ac22-4cc7-a2ca-d73ee2c98bfb" alt="get from VCS" />
+
+- Paste the URL and click clone
+
+![Paste repo url](https://github.com/ucfcs/GrowthPlus/assets/45129978/b1915eac-f165-4820-8b69-ea5cbd5de8c5)
+
+- Click file and sync project with gradle files
+
+![sync project with gradle](https://github.com/ucfcs/GrowthPlus/assets/45129978/a7bed52f-f382-4310-966e-d1b00e7a45fa)
+
+- After the syncing process is completed, click the green triangle to run the project (make sure you have an emulator already set up)
+
+![Run project](https://github.com/ucfcs/GrowthPlus/assets/45129978/d8957b6b-cf78-4da9-9db4-8e94b06116e8)
+
+- Congratulations!, the app should up and running
+
+![Running app](https://github.com/ucfcs/GrowthPlus/assets/45129978/2f8d2a5a-7046-4d92-86c7-d78ae4e4183e)
+
+## Contributing
+
+Check out [CONTRIBUTING.md](https://github.com/ucfcs/GrowthPlus/blob/julio/openSourceDocs/CONTRIBUTING.md) for the process for submitting pull requests to us and for details on our code of conduct.
+
+## Authors
+ [About the team](https://arroseplus.netlify.app/team)
```

Comment:
I changed our domain name with a more official domain:
https://growthplus4a.org/team
Can you pls update the url to this?